Zotero/Scenes/Detail/PDF/PDFCoordinator.swift

```diff
@@ -357,7 +357,7 @@ extension PDFCoordinator: PdfAnnotationsCoordinatorDelegate {
                 DDLogError("PDFCoordinator: can't render annotation image - \(error)")
                 elementProvider([])
             }
-            .disposed(by: disposeBag)
+            .disposed(by: self.disposeBag)
```

Comment:
Since using `self` for instance properties is unnecessary, unless there is one with the same name in the local scope, we should decide if we want to adhere to always prefixing w/ `self`, as part of the project style.

Reply:
I actually got an error here when using `xcbuild` during testing. I can check whether it'll still show an error here.

Anyway in the whole codebase I added `self` everywhere explicitly. I personally find it nicer because you immediately know which `disposeBag` (in this case) is being used, without having to look whether there is a local `disposeBag` instance somewhere. I know Xcode highlights them differently, but Xcode highlights are also broken most of the time. But I know most people find it annoying and even Apple in their code remove `self` everywhere. If you think that explicit `self`s are mostly unnecessary we can start reverting it. I don't feel that strongly about one or another approach.

Reply:
Oh, because github actions use Xcode 14.2, and implicit use of `self` for this case was added in 14.3!